beacon-chain/node/node.go

```diff
@@ -662,3 +669,109 @@ func (b *BeaconNode) registerArchiverService() error {
 	})
 	return b.services.RegisterService(svc)
 }
+
+func loadChainConfigFile(chainConfigFileName string) {
+	yamlFile, err := ioutil.ReadFile(chainConfigFileName)
+	if err != nil {
+		log.WithError(err).Fatal("Failed to read chain config file.")
+	}
+	// convert 0x hex inputs to fixed bytes arrays
+	lines := strings.Split(string(yamlFile), "\n")
+	for i, line := range lines {
+		if !strings.HasPrefix(line, "#") && strings.Contains(line, "0x") {
+			parts := replaceHexStringWithYAMLFormat(line)
+			lines[i] = strings.Join(parts, "\n")
+		}
+	}
+	yamlFile = []byte(strings.Join(lines, "\n"))
+	conf := params.BeaconConfig()
+	if err := yaml.Unmarshal(yamlFile, conf); err != nil {
+		log.WithError(err).Fatal("Failed to parse chain config yaml file.")
+	}
+	log.Infof("Config file values: %+v", conf)
```

Comment:
I dont think we should be logging this

Reply:
should i remove it?

Reply:
yes, its going to be noisy and confusing for the average user.

Reply:
changed to debug